### ISSUE TITLE
DynaMind-GDALModules Windows compatibiliy status

### ISSUE THREAD

**Issue by attozeta** [iut-ibk/DynaMind-ToolBox#280]
Currently the win64 build is broken for DynaMind-GDALModules (-DWITH_PLUGIN_GDAL_MODULE=On). In particular sfcgal is required and although easy to install under Linux (and probably Mac OS), this seems like a non-trivial undertaking under Windows.

Are their plans to make its build windows compatible or should we exclude it from the win build?

**Comment by christianurich** [comment]
On win32 it works fine. Had to patch it though to compile with vs2008.

**Comment by attozeta** [comment]
I made progress on win64, it now builds with vs 2013 (version 12.0) with a patched variant of sfcgal (delete operator=(...) declaration in src/Geometry.h).

**Comment by christianurich** [comment]
well done! I'm looking forward to the build.

**Comment by christianurich** [comment]
@attozeta I just saw you forked SFCGAL and I assume it's for the windows build. I have a fork that compiles with VC9.0 https://github.com/christianurich/SFCGAL which might be useful.

**Comment by attozeta** [comment]
Yeah, it is just one small change, in particular Geometry.h provides an assignment operator, however it is not used, but crashes VC12. Getting rid of it solved the problem. Where shall we push the final versions? dynamind-toolbox.org is this Wordpress page, do we want to keep it, or do we build a simple page?

**Comment by christianurich** [comment]
I would prefer building a really simple page. Updating all the websites is just to much work .....

**Comment by attozeta** [comment]
Yeah me too, I will make a new repository for a simple static page. I think it would deprecate the current dynamind-toolbox.org. So we have

- dynamind-toolbox.org:
  - a static, fancy web page
  - about
  - documentation (for users using the UI); getting started, examples, in-depth documentation
  - development (for users/developers interested in internal stuff, etc.), probably link to http://iut-ibk.github.io/DynaMind-ToolBox/
  - download: executables for windows

- http://iut-ibk.github.io/DynaMind-ToolBox/
  - the development page, as is


-
